docs/sources/developer/architecture/components/transports.md

```diff
@@ -0,0 +1,26 @@
+# Transports
+
+Transports are the final data processors in the Faro architecture. They are responsible for doing something with the
+data once it was collected by the instrumentations and processed by the internal API.
+
+By default, the core library does not provide any transports out of the box and they are either provided by wrapper
```

Comment:
`The core library does not provide any transports out of the box. They are either provided by wrapper` (or `By default, the core library does not provide any transports. They are either [...]`?